### PR TITLE
feat: even chapter buttons

### PR DESCRIPTION
**Pull request by gikf**
- [x] I have read [Chapter's contributing guidelines](https://github.com/freeCodeCamp/chapter/blob/main/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update README.md`).
- [x] My pull request targets the `main` branch of Chapter.

Ref https://github.com/freeCodeCamp/chapter/issues/1631#issuecomment-1319734507

---

- Action buttons are now even.
image
- Simplifies logic for displaying action buttons.

### CODE REVIEW

File: client/src/modules/chapters/pages/chapterPage.tsx [hunk 1]
```diff
@@ -84,20 +93,30 @@ const ChapterUserRoleWidget = ({
...
+          <CheckIcon marginRight={1} />
+          {chapterUser.chapter_role.name} of the chapter
+        </Text>
+      </Box>
+      <Box>
+        <Button onClick={LeaveChapter} width="100%">
+          Leave
+        </Button>
+      </Box>
+    </>
```
Comment (Sboonny) [inline client/src/modules/chapters/pages/chapterPage.tsx:108]:
```suggestion
    <>
      <Text data-cy="join-success" fontWeight={500}>
        <CheckIcon marginRight={1} />
        {chapterUser.chapter_role.name} of the chapter
      </Text>
      <Button onClick={LeaveChapter}>
        Leave
      </Button>
    </>
```

File: client/src/modules/chapters/pages/chapterPage.tsx [hunk 2]
```diff
@@ -84,20 +93,30 @@ const ChapterUserRoleWidget = ({
...
+    <>
+      <Box>
+        <Text fontWeight={500}>Become member of the chapter</Text>
+      </Box>
+      <Box>
+        <Button colorScheme="blue" onClick={JoinChapter} width="100%">
+          Join
+        </Button>
+      </Box>
+    </>
```
Comment (Sboonny) [inline client/src/modules/chapters/pages/chapterPage.tsx:119]:
```suggestion
    <>
      <Text fontWeight={500}>Become member of the chapter</Text>
      <Button colorScheme="blue" onClick={JoinChapter}>
        Join
      </Button>
    </>
```

File: client/src/modules/chapters/pages/chapterPage.tsx [hunk 3]
```diff
@@ -49,28 +48,38 @@ const SubscriptionWidget = ({
...
+          colorScheme="blue"
+          data-cy="subscribe-chapter"
+          onClick={() => chapterSubscribe(true)}
+          size="md"
+          width="100%"
+        >
+          Subscribe
+        </Button>
+      </Box>
+    </>
```
Comment (Sboonny) [inline client/src/modules/chapters/pages/chapterPage.tsx:82]:
```suggestion
    <>
      <Text fontWeight={500}>Follow upcoming chapter&apos;s events</Text>
      <Button
        colorScheme="blue"
        data-cy="subscribe-chapter"
        onClick={() => chapterSubscribe(true)}
      >
        Subscribe
      </Button>
    </>
```

File: client/src/modules/chapters/pages/chapterPage.tsx [hunk 4]
```diff
@@ -49,28 +48,38 @@ const SubscriptionWidget = ({
...
+        <Button
+          data-cy="unsubscribe-chapter"
+          onClick={() => chapterSubscribe(false)}
+          size="md"
+          width="100%"
+        >
+          Unsubscribe
+        </Button>
+      </Box>
+    </>
```
Comment (Sboonny) [inline client/src/modules/chapters/pages/chapterPage.tsx:65]:
```suggestion
    <>
      <Text fontWeight={500}>Unfollow upcoming chapter&apos;s events</Text>
      <Button
        data-cy="unsubscribe-chapter"
        onClick={() => chapterSubscribe(false)}
      >
        Unsubscribe
      </Button>
    </>
```

**Review by Sboonny — COMMENTED** [review]
The `Box` elements can go and we won't need to define the `size` or `width`, with the usage of grid system, because its elements will try to take as much space as possible

image

**Comment by gikf** [comment]
Thanks, I've followed example from chakra documentation, and didn't check if everything in it was necessarily required.

**Review by Sboonny — APPROVED** [review]
LGTM 🎉, thank you for your contribution to the page! 👍.

**Pull request merged**